presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -60,32 +60,20 @@ public TestLogicalPlanner()
     }
```

Comment:
Well, there is no issue ;)

I just removed tests whose the main purpose was to check if logical plan assertions are working. Instead I left test whose the main purpose is to check if logical planner return correct plan. 

Logical plan assertions are tested anyway, I do think we need special tests for that.

As you said it is a minor change in the organization of the tests ;)